ondewo/vtsi/voip.proto

```diff
@@ -240,20 +241,29 @@ message StartCallInstanceRequest {
   // initial text the sip-sim instance should say when someone calls
   string init_text = 6;
 
+  // prefix of sip name and sip container name
+  string sip_prefix = 7;
+
+  // name of the sip instance
+  string sip_name = 8;
+
+  // dictionary of sip passwords
+  google.protobuf.Struct password_dictionary = 9;
+
   // asterisk service configuration
-  ServiceConfig asterisk_config = 7;
```

Comment:
you should never change these numbers, to preserver backwards compatibility, please add the new stuff at the end